frontend/scss/components/templates/roadmap.scss

```diff
@@ -43,236 +66,279 @@
     margin: 0 auto;
   }
 
-  button.#{molecule('filter-bubble')} {
-    background: color('white');
-    color: color('silver');
-    position: relative;
-    transition: 200ms;
-    margin: 5px;
-    &.active {
-      color: color('white');
-      &.presentation {
-        background: color('cinnabar');
-      }
-      &.dynamic-personalized-content {
-        background: color('mountain-meadow');
-      }
-      &.runtime, &.tooling {
-        background: color('pickled-bluewood');
-      }
-      &.user-input, &.pwa {
-        background: color('zest');
-      }
-      &.ads {
-        background: color('studio');
-      }
-      &.amp-story {
-        background: color('buttercup');
-      }
-      &.audio-video {
-        background: color('mariner');
-      }
-      &.analytics {
-        background: color('persimmon');
-      }
-      &.email {
-        background: color('shamrock');
-      }
-      &.amp-caches {
-        background: color('lavender');
-      }
-      &.validator {
-        background: color('blue-ribbon');
-      }
+  .#{organism('roadmap')} {
+    $root: &;
+    padding: 0 15px;
+
+    @media (min-width: 768px) {
+      display: grid;
+      grid-template-columns: repeat(24,1fr);
+      grid-gap: 30px;
+      max-width: 1920px;
+      margin: auto;
+      padding: 0 2.5em;
     }
-    &:hover {
-      color: color('white');
-      &.presentation {
-        background: color('cinnabar');
-      }
-      &.dynamic-personalized-content {
-        background: color('mountain-meadow');
-      }
-      &.runtime, &.tooling {
-        background: color('pickled-bluewood');
-      }
-      &.user-input, &.pwa {
-        background: color('zest');
-      }
-      &.ads {
-        background: color('studio');
-      }
-      &.amp-story {
-        background: color('buttercup');
-      }
-      &.audio-video {
-        background: color('mariner');
-      }
-      &.analytics {
-        background: color('persimmon');
-      }
-      &.email {
-        background: color('shamrock');
+
+    &-intro {
+      margin-bottom: 0;
+
+      @media (min-width: 768px) {
+        grid-column: 3/22;
       }
-      &.amp-caches {
-        background: color('lavender');
+
+      @media (min-width: 1024px) {
+        grid-column: 8/20;
+
+        h4 {
+          display: none;
+        }
       }
-      &.validator {
-        background: color('blue-ribbon');
+    }
+
+    &-filter {
+      grid-column: 8/22;
+      margin-bottom: 50px;
+
+      .#{molecule('filter-bubble')} {
```

Comment:
Wondering why so much extra styling is required here. Isn't the same filter bubble used as on the example and components overview page?